samples/snippets/src/main/java/IdTokenFromImpersonatedCredentials_OAuth.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2022 Google Inc.
```

Comment:
I'm not going to review this snippet right now. We need to decide how and where to incorporate it. Right now, it doesn't really fit the flow in either the authentication docs or the IAM docs. But it's good we have it. Can you put it somewhere where we can return to it, but it's not yet visible to users?

Reply:
We can leave it as such in the repo and not add a region tag. That way it'll not be picked up by the snippet bot and hence will not be visible in cgc. wdyt?